docs/compilers/CSharp/Runtime Async Design.md

```diff
@@ -47,6 +47,10 @@ public static class RuntimeHelpers
     public static void Await(ValueTask task);
     [MethodImpl(MethodImplOptions.Async)]
     public static T Await<T>(ValueTask<T> task);
+    [MethodImpl(MethodImplOptions.Async)]
```

Comment:
These methods are not currently async. I don’t believe we’ve finalized the decision on whether or not they will be.